### DISCUSSION TITLE
Neutral Helper type (teamside =0)

### DISCUSSION THREAD

**Discussion by Noobie613** [ikemen-engine/Ikemen-GO#1764] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

the problem with current helpers is they can't be hit or interact with (yeah i know about affectteam in hitdef but i want more) by the player or your partner in simul mode

### Describe the solution you'd like

So i see u guys add teamside = 0 to attached char for stage. I wonder u guys can add this feature to helper type too? Or something like a new collision box that both team can hit and interact with? 
Thanks guys

### Search terms

Teamside clsn

**Comment by potsmugen** [comment, marked as answer]
You can use `ModifyChar` to change teamside to 0. But it should be done with caution because the code may not expect teamside changes in some places yet.

**Comment by Noobie613** [comment]
Thanks for your reply. Well i only just want the helper to get the teamside. Anyway keep up your good work
